bazel.hcl

```diff
@@ -1,7 +1,7 @@
 description = "Bazel is an open-source build and test tool similar to Make, Maven, and Gradle."
 binaries = ["bazel"]
 
-darwin {
+platform "darwin" "amd64" {
```

Comment:
You'll have to remove `amd64`, otherwise this won't work on M1.

Reply:
I intentionally left this out, but I guess I didn't realize this could be depending on Rosetta before -- was that the case?

Reply:
This is the case yes.